assets/aml-benchmark/components/inference-postprocessor/spec.yaml

```diff
@@ -0,0 +1,94 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+type: command
+
+name: inference_postprocessor
+display_name: Inference Postprocessor
+description: Inference Postprocessor
+version: 0.0.19
+is_deterministic: true
+
+inputs:
+  prediction_dataset:
+    type: uri_file
+    description: >-
+      A file that contains predicted values
+    optional: False
+  prediction_column_name:
+    type: string
+    description: >-
+      Key in prediction dataset that contains predictions.
+    optional: False
+  ground_truth_dataset:
+    type: uri_file
+    description: >-
+      A file that contains the ground truth
+    optional: True
+  ground_truth_column_name:
+    type: string
+    description: >-
+      Key in ground truth dataset that contains ground truth. If ground_truth_dataset is given, then,
+      this is required input.
+    optional: True
+  separator:
+    type: string
+    description: >-
+      The separator used in few_shot patterns. One common example is "###".
+      If provided, response will be split on this separator, and only the first part will be used.
+      Example: "This is the first part ### This is the second part" will result in
+      "This is the first part".
+    optional: True
+  regex_expr:
+    type: string
+    description: >-
+      A regular expression to extract the answer from the inference results. The pattern
+      must contain a group to be extracted. The first group and the first match will be used. 
+      Example: "\n\nThe answer is: (\d)."
+    optional: True
+  strip_prefix:
+    type: string
+    description: >-
+      Characters to remove from the beginning of the extracted answer.
```

Comment:
For the string that we provide, is that string removed from the beginning or is the string used as a set of characters and we remove characters as long as it exists in the set? Can you share an example for this parameter?

Reply:
A string ( can be like this "### 32") to remove if it exists.

Reply:
This is also a parameter in Babel's component.